utils/nctl/sh/assets/setup.sh

```diff
@@ -53,17 +51,17 @@ function _set_chainspec() {
     # Set directory.
     mkdir $1/chainspec
 
-    # Set config params.
-    GENESIS_NAME=casper-net-$2
-    GENESIS_TIMESTAMP=$(get_genesis_timestamp $3)
-    HIGHWAY_GENESIS_ERA_START_TIMESTAMP=$(get_genesis_timestamp $3)
-
     # Set config.
     path_config=$1/chainspec/chainspec.toml
-    cp $NCTL/templates/chainspec.toml $path_config
-    sed -i "s/{GENESIS_NAME}/$GENESIS_NAME/g" $path_config > /dev/null 2>&1
-    sed -i "s/{GENESIS_TIMESTAMP}/$GENESIS_TIMESTAMP/g" $path_config > /dev/null 2>&1
-    sed -i "s/{HIGHWAY_GENESIS_ERA_START_TIMESTAMP}/$HIGHWAY_GENESIS_ERA_START_TIMESTAMP/g" $path_config > /dev/null 2>&1
+    cp $NCTL_CASPER_HOME/resources/local/chainspec.toml $path_config
+
+    # Set config setting: genesis.name.
+    GENESIS_NAME=casper-net-$2
+    sed -i "s/casper-example/$GENESIS_NAME/g" $path_config > /dev/null 2>&1
```

Comment:
Could matching on "casper-example" break in the future? If it were to get renamed, we'd no longer do the substitution properly

Reply:
Yeah it is a little brittle.  Might try to improve it - on another day.